packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -172,6 +173,12 @@ class CupertinoPageRoute<T> extends PageRoute<T> {
 
   @override
   Color get barrierColor => null;
+  
+  /// Allows you to set opaque to false to prevent route reconstruction. 
+  /// The default behavior is true, and declaring false can have side effects. 
+  /// Use caution and awareness that rebuilding is necessary to properly display the back button.
```

Comment:
This doc doesn't quite follow our style guide.  The first sentence should be on its own, and describe in one line what this does.  I'm not sure we should document this as preventing route reconstruction - it seems like it's more about whether the route is opaque or not.  Reconstruction is about an optimization that might happen if the route if opaque.

The second lines should be expanded a bit more - what does it mean for this to be true or false? Why do I care?